AutomatedTesting/Gem/PythonTests/Physics/tests/force_region/ForceRegion_NoQuiverOnHighLinearDampingForce.py

```diff
@@ -83,26 +81,26 @@ def __init__(self, name):
         @property
         def position(self):
             # type () -> Vector3
-            return azlmbr.components.TransformBus(azlmbr.bus.Event, "GetWorldTranslation", self.id)
+            return azlmbr.components.TransformBus(azlmbr.bus.Event, GET_WORLD_TRANSLATION, self.id)
 
         @property
         def velocity(self):
             # type () -> Vector3
-            return azlmbr.physics.RigidBodyRequestBus(azlmbr.bus.Event, "GetLinearVelocity", self.id)
+            return azlmbr.physics.RigidBodyRequestBus(azlmbr.bus.Event, GET_LINEAR_VELOCITY, self.id)
 
         @property
         def is_moving_up(self):
             # type () -> bool
```

Comment:
seems to me like the entire class Entity here should be in a helper as class PhysicsEntity(EditorEntity):
EditorEntity from editor_entity_utils.py
this gives you lots of basic entity utilities and then you can add all these reusable physics functions and more.
usage would be slightly different in that you would then call PhysicsEntity.find_editor_entity(name) to construct a rather than the current Entity(name).
just a though not saying this needs changing

Reply:
This is something I considered too. 

I'm looking at these tests and see great refactoring opportunities to improve the code throughout the test code base, but I also don't want to go down the rabbit hole quite yet since there's other refactors of the physics code already being considered where this could be added in.

Reply:
editor_entity_utils.py EditorEntity already has transform bus helpers. you might even be able to just add physics bus to it.
    def get_world_translation(self) -> azlmbr.math.Vector3:
        """
        Gets the world translation of the entity
        """
        return azlmbr.components.TransformBus(azlmbr.bus.Event, "GetWorldTranslation", self.id)